src/style/style.css

```
@@ -0,0 +1,139 @@
+html {
+  font-size: 10px;
+}
+
+body {
+  font-size: 1.6rem;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 2.6rem;
+  font-family: "Open Sans", "Verdana", sans-serif;
+}
+
+.virtual-keyboard-wrapper {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  max-width: 704px;
+}
+
+.header-wrapper {
+  text-align: center;
+}
+
+.header {
+  font-size: 2.4em;
+  font-family: "Roboto", "Verdana", sans-serif;
+  font-weight: 300;
+  color: #585858;
+}
+
+.description {
+  font-size: 1em;
+  line-height: 1em;
+  margin: 10px 0;
+}
+
+.textarea {
+  display: block;
+  overflow-y: auto;
+  width: 100%;
+  height: 200px;
+  margin: 10px auto 10px;
+  padding: 10px;
+  resize: none;
+  font-size: 1em;
+  color: #444444;
+  outline: none;
+}
+
+.keyboard {
+  border: 1px solid #666666;
+  border-radius: 5px;
+  background-color: #eeeeee;
+}
+
+.keyboard--row {
+  display: flex;
+  flex-wrap: nowrap;
+  margin: 2px;
+}
+
+.keyboard--btn {
+  display: block;
+  position: relative;
+  overflow: hidden;
+  width: 40px;
+  height: 40px;
+  margin: 3px;
+  border-radius: 5px;
+  line-height: 4rem;
+  font-size: 1.6rem;
+  text-align: center;
+  background-color: #afafaf;
+  color: #ffffff;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+.keyboard--btn:hover {
+  background-color: #34bdd1;
+}
+
+.keyboard .keyboard--row > :not(.symbol) {
+  background-color: #808080;
+  font-style: bold;
+}
+
+.keyboard .keyboard--row > :not(.symbol):hover {
```

Too complicated selector. Try to think how to simplify it

---

Replaced to:
```suggestion
.keyboard--row > :not(.symbol)
```
In another case, I will have to add extra classes to the elements, and the logic of their definition in the button generation method

---

fixed and refactored